templates/CRM/common/CMSPrint.tpl

```diff
@@ -40,7 +40,7 @@
 <div id="crm-main-content-wrapper">
   {include file="CRM/common/status.tpl"}
   {crmRegion name='page-body'}
-    {if isset($isForm) and $isForm and isset($formTpl)}
+    {if $isForm and $formTpl}
```

Comment:
The logic isn't exactly the same but I would say the "before" isn't completely sensible anyway.